test/cli-tests/tests/it/auth.rs

```diff
@@ -0,0 +1,50 @@
+use soroban_test::{TestEnv, Wasm};
+
+const WASM: &Wasm = &Wasm::Release("soroban_auth_contract");
+
+const ACC1: &str = "GA6S566FD3EQDUNQ4IGSLXKW3TGVSTQW3TPHPGS7NWMCEIPBOKTNCSRU";
```

Comment:
Please write out.  No need to be so terse in an example test.

Reply:
done in d6bf4f8